Resources/Prototypes/Entities/Constructible/Piping/gascanisters.yml

```diff
@@ -1,16 +1,25 @@
 - type: entity
   abstract: true
   id: GasCanisterBase
-  placement:
-    mode: SnapgridCenter
+  parent: BaseConstructibleDynamic
   components:
-  - type: Clickable
   - type: InteractionOutline
-  - type: Physics
   - type: SnapGrid
     offset: Center
   - type: Sprite
   - type: Damageable
+  - type: Physics
+    mass: 25
+    bodyType: Dynamic
+    fixtures:
+      - shape:
+          !type:PhysShapeAabb
+            bounds: "-0.5,-0.25,0.5,0.25"
+        mask:
+          - SmallImpassable
+        layer:
+          - Opaque
+          - MobImpassable
```

Comment:
SmallImpassable for mask is wrong, should probably be MobImpassable (if we're using MobImpassable as medium). Also it likely needs SmallImpassable and VaultImpassable layers.